lib/matplotlib/artist.py

```diff
@@ -1276,7 +1276,9 @@ def properties(self):
                 continue
 
             try:
-                val = func()
+                with warnings.catch_warnings():
+                    warnings.simplefilter('ignore')
```

Comment:
Followed the directions here

https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings